packet_forwarder/src/lora_pkt_fwd.c

```diff
@@ -129,6 +130,7 @@ License: Revised BSD License, see LICENSE.TXT file include in the project
 /* signal handling variables */
 volatile bool exit_sig = false; /* 1 -> application terminates cleanly (shut down hardware, close open files, etc) */
 volatile bool quit_sig = false; /* 1 -> application terminates without shutting down the hardware */
+volatile bool reload_sig = false; /* 1 -> application re-opens logs/PCAP output */
```

Comment:
I’m guessing you’re just following the existing code, but these should all be atomics. Somehow I forgot it never noticed the (incorrect) use of volatile here